rsciio/bruker/_api.py

```diff
@@ -499,7 +499,9 @@ def __init__(self, spectrum):
                 "ElevationAngle": xrf_header_dict["ExcitationAngle"],
             }
         # USED:
-        self.hv = self.esma_metadata["PrimaryEnergy"]
+        self.hv = self.esma_metadata.get("PrimaryEnergy", None)
+        if self.hv is None:
+            _logger.warning("The beam energy couldn't be found.")
```

Comment:
Waiting for a smaller file from https://github.com/hyperspy/rosettasciio/issues/323#issuecomment-2430364313.